src/js/components/help/helpContent.jsx

```diff
@@ -62,6 +62,15 @@ export default class HelpContent extends React.Component {
     }
 
     render() {
+        let membership = '';;
```

Comment:
Also, when setting something to be empty (not show up), it's better to set it to null. If you set it to an empty string React inserts a <span></span> there anyway. If you set it to null it doesn't insert anything, which is cleaner and better overall (also potentially less problematic for accessibility issues).